.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 version: 2.1
 
-parameters:
+params:
```

Comment:
?

Reply:
that's the CI fix; it was just merged into master. the diff should have gone away